.github/workflows/opened-issues-triage.yml

```diff
@@ -10,6 +10,6 @@ jobs:
     steps:
       - uses: alex-page/github-project-automation-plus@v0.3.0
         with:
-          project: "Velero Support Triage Board"
-          column: "Needs triage"
-          repo-token: ${{ secrets.GH_TOKEN }}
+          project: "Velero Support Board"
+          column: "New"
+          repo-token: ${{ secrets.GitHub_TOKEN }}
```

Comment:
Is this change in token necessary? I can't see it in the list of secrets for the repo.

Reply:
@zubron It should be all caps so I'll fix that, but it's a generated token vs one that's associated with anyone's account.

See https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

Reply:
Ah, okay! I thought there was a reason that we couldn't use that before, but I think that might have been because those tokens wouldn't work with org level boards.

Reply:
Correct, `GITHUB_TOKEN` is repository-scoped only, it can't access org-level stuff.